packages/cli/src/lib/config/configRenderer.ts

```diff
@@ -0,0 +1,22 @@
+import {cli} from 'cli-ux';
```

Comment:
Eventually we could have different method to render the config (similar to sfdx).
Meaning we could have JSON and table formats.